exporter/zipkinexporter/config.go

```diff
@@ -27,5 +27,8 @@ type Config struct {
 	URL    string `mapstructure:"url"`
 	Format string `mapstructure:"format"`
 
+	// Whether resource labels from TraceData are to be included in Span
+	ExportResourceLabels string `mapstructure:"export_resource_labels"`
```

Comment:
Why not make this a `bool`?

Reply:
Good point, I thought that the case with not-set value would not be handled but `mapstructure` is smart enough to handle this when `*bool` is used